imod_coupler/drivers/ribametamod/ribametamod.py

```diff
@@ -137,6 +137,9 @@ def initialize(self) -> None:
             self.ribasim.initialize(ribasim_config_file)
         if self.has_metaswap:
```

Comment:
The `couple` function is rather long now.

Could you rewrite it slightly, so that it essentially reads:

```python
def couple(self):
        self.max_iter = self.mf6.max_iter()
        mf6_flowmodel_key = self.coupling.mf6_model
        self.mf6_head = self.mf6.get_head(mf6_flowmodel_key)

        if self.has_ribasim:
            self.couple_ribasim()

        if self.has_metaswap:
            self.couple_metaswap()
```